.gitignore

```diff
@@ -6,6 +6,7 @@ coverage.*
 tools
 build
 .vscode
+/vendor/
```

Comment:
Not as part of this PR, please - thanks!

Reply:
good idea. removing. thanks